crawlers/mooncrawl/mooncrawl/contract/cli.py

```diff
@@ -0,0 +1,177 @@
+import argparse
+import json
+import logging
+import time
+from typing import Optional
+
+from moonstreamdb.db import yield_db_session_ctx
+from sqlalchemy.orm.session import Session
+from web3 import Web3
+
+from .deployment_crawler import ContractDeploymentCrawler, MoonstreamDataStore
+from ..ethereum import connect
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+SLEEP_TIME = 5 * 60
+
+
+def runc_crawler_asc(
```

Comment:
Would it be possible to consolidate `run_crawler_asc` and `run_crawler_desc` by creating a `run_crawler` function with an additional argument `order: CrawlOrder`? Here:

```
from enum import Enum

class CrawlOrder(Enum)
    asc = "asc"
    desc = "desc"
```

Reply:
Also, why not make these methods of `ContractDeploymentCrawler`?

Reply:
It was as one function before, but it was really hard to read (too many nested `if`s to check `asc` or `desc` each time)
After it became too hard to understand code, i split it

About implementing as functions of `ContractDeploymentCrawler`, yes agree it would be better